runtime/src/lib.rs

```diff
@@ -190,36 +233,38 @@ pub const DAYS: BlockNumber = HOURS * 24;
 pub const CONTRACTS_DEBUG_OUTPUT: bool = true;
```

Comment:
@athei The output is only printed if the node is started with `-lruntime::contracts=debug`, so it should be fine to keep this `true` ‒ right?

Reply:
Yes. It should be set to true. There is no real reason to ever pass `false` there from the RPC function. The only thing I can imagine is that you are running a public RPC node and want to reduce load or prevent DoS vectors.

In order to make this possible we should not hardcode this in the runtime but pull the value from the client. Basically make this configurable on the command line. But this is not important right now.